datasets/wise-unwise.yaml

```
@@ -0,0 +1,32 @@
+Name: "Unofficial, Unblurred Coadds of the Wide-field Infrared Survey Explorer (unWISE)"
```

```suggestion
Name: "Unblurred Coadds of the Wide-field Infrared Survey Explorer (unWISE)"
```

I don't see the term 'unofficial' in the original unWISE paper or the IRSA website, but am not as familiar with all this documentation as you are.    Also, 'unofficial' sounds somehow 'not as good', but that is not the point.